docs/managed_vms/endpoints/app.go

```diff
@@ -51,8 +53,22 @@ func echoHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write(b)
 }
 
+// corsHandler wraps a HTTP handler and applies the appropriate responses.
+func corsHandler(h func(http.ResponseWriter, *http.Request)) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
+		if r.Method == "OPTIONS" {
+			w.Header().Set("Access-Control-Allow-Headers", "Authorization")
+			return
+		}
+		h(w, r)
+	}
+}
+
 // authInfoHandler reads authentication info provided by the Endpoints proxy.
 func authInfoHandler(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
And it does. I forgot to revert this line. We don't want it for the other handlers.